source/extensions/transport_sockets/tls/context_impl.h

```diff
@@ -169,7 +169,7 @@ class ContextImpl : public virtual Envoy::Ssl::Context {
                      const Ocsp::OcspResponseWrapper* ocsp_response) const;
 
   struct TlsContext {
-    // Each certificate specified for the context has its own SSL_CTX. SSL_CTXs
+    // Each certificate specified for the context has its own SSL_CTX. `SSL_CTXs`
```

Comment:
Drive by: SSL_CTX being in the dictionary but SSL_CTXs requiring ``'s around it seems a bit strange.

Reply:
While removing CTXs from the dictionary SSL_CTXs requires ``'s around it.

Reply:
Looks a little weird, but I'm guessing it'll get cleaner as more of the dictionary is cleaned out.